### ISSUE TITLE
ValueError: Unknown options: analyze (Sqlite backend)

### ISSUE THREAD

**Issue by Archmonger** [jazzband/django-silk#489]
Django Silk `ANALYZE_QUERIES` does not work with Sqlite3 when using Django 3.2.6.

Django silk fetched via `pip install -e git+https://github.com/jazzband/django-silk.git#egg=django-silk`.

Django-silk latest commit: [`7f03cfef0b554af541dc53fc6d5bee850469098a`](https://github.com/jazzband/django-silk/commit/7f03cfef0b554af541dc53fc6d5bee850469098a).

```python
# settings.py
SILKY_AUTHENTICATION = True
SILKY_AUTHORISATION = True 
SILKY_ANALYZE_QUERIES = True # This appears to result in the problem taking place.
SILKY_PYTHON_PROFILER = True
SILKY_PYTHON_PROFILER_BINARY = True
```

This issue is not present on Django 3.1 using Django-silk 4.1

### Traceback
```cmd
Traceback (most recent call last):
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\asgiref\sync.py", line 482, in thread_handler
    raise exc_info[1]
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\core\handlers\exception.py", line 38, in inner
    response = await get_response(request)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\core\handlers\base.py", line 233, in _get_response_async
    response = await wrapped_callback(request, *callback_args, **callback_kwargs)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\asgiref\sync.py", line 444, in __call__
    ret = await asyncio.wait_for(future, timeout=None)
  File "C:\Users\username\AppData\Local\Programs\Python\Python39\lib\asyncio\tasks.py", line 442, in wait_for
    return await fut
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\asgiref\current_thread_executor.py", line 22, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\asgiref\sync.py", line 486, in thread_handler
    return func(*args, **kwargs)
  File "c:\users\username\documents\repositories\conreq\.venv\src\django-silk\silk\profiling\profiler.py", line 158, in wrapped_target
    result = target(*args, **kwargs)
  File "C:\Users\username\Documents\Repositories\Conreq\conreq\core\base\views.py", line 86, in landing
    config_needed = configure(request)
  File "C:\Users\username\Documents\Repositories\Conreq\conreq\core\base\views.py", line 21, in configure
    conreq_config = ConreqConfig.get_solo()
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\solo\models.py", line 55, in get_solo
    obj, created = cls.objects.get_or_create(pk=cls.singleton_instance_id)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\manager.py", line 85, in manager_method
    return getattr(self.get_queryset(), name)(*args, **kwargs)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\query.py", line 581, in get_or_create
    return self.get(**kwargs), False
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\query.py", line 431, in get
    num = len(clone)
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\query.py", line 262, in __len__
    self._fetch_all()
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\query.py", line 1324, in _fetch_all
    self._result_cache = list(self._iterable_class(self))
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\models\query.py", line 51, in __iter__
    results = compiler.execute_sql(chunked_fetch=self.chunked_fetch, chunk_size=self.chunk_size)
  File "c:\users\username\documents\repositories\conreq\.venv\src\django-silk\silk\sql.py", line 81, in execute_sql
    query_dict['analysis'] = _explain_query(q, params)
  File "c:\users\username\documents\repositories\conreq\.venv\src\django-silk\silk\sql.py", line 33, in _explain_query
    prefix = connection.ops.explain_query_prefix(
  File "C:\Users\username\Documents\Repositories\Conreq\.venv\lib\site-packages\django\db\backends\base\operations.py", line 686, in explain_query_prefix 
    raise ValueError('Unknown options: %s' % ', '.join(sorted(options.keys())))
```

### Related LOC
Issue appears to be related to django-silk/silk/sql.py:33
```python
            prefix = connection.ops.explain_query_prefix(
                analyze = True
            )
```

**Comment by Archmonger** [comment]
@tiltec Can you confirm if you can replicate this issue?

**Comment by tiltec** [comment]
Just tried it, seems to work here (Django 3.2.6, django-silk latest commit).

I can make a request and then see the query plan with execution time in Silk:
![image](https://user-images.githubusercontent.com/4410802/129768717-c851d0f5-4ac6-453f-b0ff-d0dfdceca0e1.png)

**Comment by albertyw** [comment]
It would be preferable to automate this by modifying https://github.com/jazzband/django-silk/blob/master/.github/workflows/test.yml to include setting up a sqlite database and running tests against it.  That would help to prove compatibility.

**Comment by TheVinhLuong102** [comment]
I am having this very same issue. It's not a big deal for production apps using PG/MySQL, but it is a pain for the prototyping phase, which typically involves SQLite.

**Comment by marius-mather** [comment]
> It would be preferable to automate this by modifying https://github.com/jazzband/django-silk/blob/master/.github/workflows/test.yml to include setting up a sqlite database and running tests against it. That would help to prove compatibility.

It looks like the project is already set up to test against multiple Python versions and databases, the problem is the existing tests all seem to mock queries so nothing hits the actual DB. I'll try to submit a PR with tests that actually call out to the DB.

**Comment by marius-mather** [comment]
I think this should be closed via #493
